manifests/config.pp

```diff
@@ -164,7 +164,7 @@
     # }
 
     # Generate Elasticsearch config
-    $data = merge(
+    $data = stdlib::merge(
```

Comment:
I would really like to release a new version that allow 9.x before we start requiring 9.x
This makes it a bit easier to upgrade without breaking dependencies.